src/main/java/net/fabricmc/tinyremapper/TinyRemapper.java

```diff
@@ -714,11 +711,11 @@ private void handleConflicts() {
 		}
 	}
 
-	public void apply(final BiConsumer<String, byte[]> outputConsumer) {
+	public void apply(final BiConsumer<VersionedName, byte[]> outputConsumer) {
 		apply(outputConsumer, (InputTag[]) null);
 	}
 
-	public void apply(final BiConsumer<String, byte[]> outputConsumer, InputTag... inputTags) {
+	public void apply(final BiConsumer<VersionedName, byte[]> outputConsumer, InputTag... inputTags) {
```

Comment:
Need to keep `BiConsumer<String, byte[]>` versions or this is a breaking change to all users

Reply:
fixed